components/create_post/create_post.tsx

```diff
@@ -1316,6 +1329,61 @@ class CreatePost extends React.PureComponent<Props, State> {
         });
     }
 
+    completePostTip = () => {
+        this.props.actions.savePreferences(
+            this.props.currentUserId,
+            [{
+                user_id: this.props.currentUserId,
+                category: Constants.Preferences.TUTORIAL_STEP,
+                name: this.props.currentUserId,
+                value: (Constants.TutorialSteps.POST_POPOVER + 1).toString(),
+            }],
+        );
+    }
+
+    renderPrewrittenMessages() {
+        if (this.props.prewrittenMessages !== PrewrittenMessagesTreatments.AROUND_INPUT || this.props.tutorialStep !== Constants.TutorialSteps.POST_POPOVER) {
+            return null;
+        }
+
+        let id = t('create_post.prewritten.around_input.team');
+        let defaultMessage = '**Send your first message** to your team';
+        if (this.props.currentChannel.type === 'D') {
+            if (this.props.currentChannel.teammate_id === this.props.currentUserId) {
+                id = t('create_post.prewritten.around_input.self');
+                defaultMessage = '**Send your first message** to yourself';
+            } else {
+                id = t('create_post.prewritten.around_input.dm');
+                defaultMessage = '**Send your first message** to your teammate';
+            }
+        }
+        return (
+            <div>
```

Comment:
is this div used to layout the internal structure, or just to accomplish the react rule of only render one tag? if so, maybe you could use a React.Fragment `<></>` instead.